app/views/register/nonTaxable/TrusteesInfoView.scala.html

```diff
@@ -0,0 +1,97 @@
+@*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import controllers.register.routes._
+
+@this(
+    main_template: MainTemplate
+)
+
+@(draftId: String)(implicit request: Request[_], messages: Messages)
+
+@main_template(
+    title = messages("trusteesInfo.nonTaxable.title")
+    ) {
+
+    @components.back_link()
+
+    @components.heading_with_caption("trusteesInfo.nonTaxable")
+
+    <h2>@messages("trusteesInfo.nonTaxable.subheading1")</h2>
+
+    <p>@messages("trusteesInfo.nonTaxable.paragraph1")</p>
+
+    <p>@messages("trusteesInfo.nonTaxable.paragraph2")</p>
+
+    <h2>@messages("trusteesInfo.nonTaxable.subheading2")</h2>
+
+    <p>@messages("trusteesInfo.nonTaxable.paragraph3")</p>
+
+    @components.bullet_list("trusteesInfo.nonTaxable",
+        List(
+            "bulletpoint1",
+            "bulletpoint2",
+            "bulletpoint3",
+            "bulletpoint4",
+            "bulletpoint5",
+            "bulletpoint6",
+            "bulletpoint7",
+            "bulletpoint8"
+        )
+    )
+
+    <p>@messages("trusteesInfo.nonTaxable.paragraph4")</p>
+
+    <p>@messages("trusteesInfo.nonTaxable.paragraph5")</p>
+
+    <p>@messages("trusteesInfo.nonTaxable.paragraph6")</p>
+
+    <h2>@messages("trusteesInfo.nonTaxable.subheading3")</h2>
+
+    <p>@messages("trusteesInfo.nonTaxable.paragraph7")</p>
+
+    @components.bullet_list("trusteesInfo.nonTaxable",
+        List(
+            "bulletpoint9",
+            "bulletpoint10",
+            "bulletpoint11",
+            "bulletpoint12",
+            "bulletpoint13",
+            "bulletpoint14"
+        )
+    )
+
+    <p>@messages("trusteesInfo.nonTaxable.paragraph8")</p>
+
+    <details class="govuk-details" data-module="govuk-details">
+        <summary class="govuk-details__summary">
+        <span class="govuk-details__summary-text">
+          @messages("trusteesInfo.nonTaxable.details")
+        </span>
+        </summary>
+        <div class="govuk-details__text">
```

Comment:
The markup you have for this page is:

```
- h1
- h2
- h2
- h4
- h4
```
This won't pass an a11y assessment and would be flagged up on an Axe test. Can you revise the headings inside of the `<details>` to either just be bold paragraphs or h3's.
Also, can you ensure the content is wrapped inside `<p>` tags inside the details

Reply:
@adamconder the design system markup does not use <p> tags inside the <details>:
https://design-system.service.gov.uk/components/details/

Reply:
@jassalrichy I agree it doesn't but I think that is because it is simple 1 paragraph of text inside therefore the details is sufficient?
Do we need to saying we have further content. Just semantically the paragraph will assist screen readers that its separate content etc.